src/interval.jl

```diff
@@ -288,18 +295,26 @@ function Base.union!(intervals::Union{AbstractVector{<:Interval}, AbstractVector
     return intervals
 end
 
+"""
+    superset(intervals::AbstractArray{<:AbstractInterval}) -> Interval
+
+Create a single interval which encompasses all of the provided intervals.
```

Comment:
Perhaps it would be worthwhile to mention that this is the smallest such interval?